src/components/TodoComponents/Todo.js

```diff
@@ -1,2 +1,27 @@
 import React from 'react';
+//////////////////////////////////////
+// CODE FROM TUTORIAL
 
+class Todo extends React.Component {
+
+    createTasks = item => {
+        return <li key={item.key} onClick={() => this.props.completeItem(item)} > {item.text} </li>
+    }
```

Comment:
I would also recommend setting a different class for completed items (to visually indicated that they were marked as such). The class could be something like:
```
.completed-todo {
  text-decoration: line-through; /* Adds a line that crosses out text */
}
```

You can then set this class on an element using a ternary expression (which is like an expression form of an `if-else` statement). For example:
```
<li className={item.completed ? "complete-todo" : "incomplete-todo"}>
```

_(note that `item.completed` has the same result as `item.completed === true`)_